run_race.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+import queue
+import runner_class
+import os
+
+
+class RunRace:
+    def __init__(self, runners: set, race_length: int):
+        self.runners = runners
+        self.race_length = race_length
+        self.places = queue.Queue(len(runners))
+
+    def start_race(self):
+        loop = asyncio.get_event_loop()
+        for current_runner in self.runners:
+            loop.create_task(
+                current_runner.run(self.race_length - current_runner.start_point, self.__print_current_stat,
+                                     self.__register_place_of_runner))
+        pending = asyncio.Task.all_tasks()
+        results = loop.run_until_complete(asyncio.gather(*pending))
+        print("the winner is {}".format(self.places.queue.popleft().name))
+        loop.close()
+
+    def __print_current_stat(self):
+        os.system('cls' if os.name == 'nt' else "printf '\033c'")
```

Comment:
Never use os.system in production code
the code will execute differently in each operating system

And btw, what is the use of this line? I removed it and the output stayed the same...

Reply:
I am doing it to delete the preview's terminal messages. 
it's not working at pycharm it's work in a regular terminal.
it's work better like that but I understand what you say